sql/share/errmsg-utf8.txt

```diff
@@ -7880,7 +7880,7 @@ ER_VERS_HISTORY_LOCK
         eng "Versioned SELECT write-locking of history rows"
 
 ER_VERS_NO_TRX_ID
-        eng "TRX_ID %lu not found in VTQ"
+        eng "TRX_ID %lu not found in TRT"
```

Comment:
It better to use full table name for ease of reading.

Reply:
Fixed.